templates/home.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <body>
+   <div class="home">
+     <h1>Dear user!</h1>
+     <p>Welcome to Coonect where you can start amazing initiatives!</p>
+   </div>
+  </body>
+</html><form method="POST" ACTION="">
```

Comment:
this does not have to be a form. What you want to do here is:
- Add a link to register (that will take you to the register page)
- Add a link to login (that will take you to the login page)
you can style the links later with css to make them look like buttons :)

so something like:
```html
<a href="/register">Register</a>
<a href="/login">Login</a>
```
Don't forget to add the corresponding `GET` pages for each! :)